api/envoy/config/filter/http/ext_authz/v2alpha/ext_authz.proto

```diff
@@ -94,7 +94,7 @@ message AuthorizationRequest {
   // be forwarded *from the filter* to the authorization server. Note that *Content-Length*,
   // *Authority*, *Method*, *Path* and *Authorization* are always dispatched to the authorization
   // server by default. The message will not contain body data and the *Content-Length* will be set
-  // to zero.
+  // to zero. String matcher should be lower case.
```

Comment:
I don't think this matters since it gets loaded into a lower case string before matching, right? Same below?

Reply:
No, these objects get passed straight into `Matchers::StringMatcher's` constructor. I think we need a case-insensitive StringMatcher. Wdyt?

Reply:
Hmm, I see. Yes, that would be optimal. Do you want to do that in a follow up or work on that here?

Reply:
I address it here. I've been put out some fires lately, but I will make sure that this gets done by today.

Reply:
I came up with a solution, just need to write some tests.